src/touchhandler/TouchHandler.cpp

```diff
@@ -0,0 +1,76 @@
+#include "TouchHandler.h"
+
+using namespace Pinetime::Controllers;
+
+TouchHandler::TouchHandler(Drivers::Cst816S& touchPanel, Components::LittleVgl& lvgl) : touchPanel {touchPanel}, lvgl {lvgl} {
+}
+
+void TouchHandler::CancelTap() {
+  if (info.touching) {
+    isCancelled = true;
+    lvgl.SetNewTouchPoint(-1, -1, true);
+  }
+}
+
+Pinetime::Drivers::Cst816S::Gestures TouchHandler::GestureGet() {
+  auto returnGesture = gesture;
+  gesture = Drivers::Cst816S::Gestures::None;
+  return returnGesture;
```

Comment:
This could be simplified as `return std::exchange(gesture, Drivers::Cst816S::Gestures::None);`

Reply:
It could be argued whether this is simpler or not. Anyone can read the current code, but I must not be the only one who's never heard of `std::exchange()` before. I can change it though if this is important to comply with the "Modern C++" requirement.

Reply:
Up to you ultimately, just a suggestion. I just like to mention things like these because as you said yourself there's lots of useful things in the standard library that many people aren't familiar with. I personally think that `std::exchange` is nicer/simpler than the equivalent code to do it manually but YMMV.